src/adapters/appnexusAst.js

```diff
@@ -76,7 +85,27 @@ function AppnexusAstAdapter() {
           tag.keywords = getKeywords(bid.params.keywords);
         }
 
-        if (bid.mediaType === 'video') {tag.require_asset_url = true;}
+        if (bid.mediaType === 'native') {
+          tag.ad_types = ["native"];
+
+          if (bid.nativeParams) {
+            tag.native = {};
+            const nativeRequest = {};
+
+            // map standard native asset identifier to what /ut expects
+            Object.keys(bid.nativeParams).forEach(key => {
+              let requestKey = NATIVE_MAPPING[key] && NATIVE_MAPPING[key].serverName || NATIVE_MAPPING[key] || key;
```

Comment:
It's required because some of the standard native assets prebid defines don't translate exactly to v3 endpoint parameters. v3 looks for `description` request parameter rather than `body` for example. Agreed this code is confusing though, I'll refactor to be more clear